{
  Assert.assertTrue(invoke("toTitlecase") instanceof EvalError);
  Assert.assertTrue(invoke("toTitlecase","one","two","three") instanceof EvalError);
}
